lib/ex_machina.ex

```diff
@@ -4,6 +4,8 @@ defmodule ExMachina do
 
   ## Examples
 
+  Put this somewhere like `test/support/ex_machina.ex`
```

Comment:
Not sure about that. It is more specific, but longer too. What did you dislike about "Put this somewhere like `test/support/ex_machina.ex`?